core/services/relay/evm/event_binding.go

```diff
@@ -227,32 +227,56 @@ func (e *eventBinding) getLatestValueWithFilters(
 		return err
 	}
 
-	fai := filtersAndIndices[0]
-	remainingFilters := filtersAndIndices[1:]
-
-	logs, err := e.lp.IndexedLogs(ctx, e.hash, e.address, 1, []common.Hash{fai}, confs)
+	// Create limiter and filter for the query.
+	limiter := query.NewLimitAndSort(query.CountLimit(1), query.NewSortBySequence(query.Desc))
```

Comment:
As we only want the latest, we make `ORDER BY SEQUENCE DESC LIMIT 1`. 

Note: Picked by sequence as it was the most used one in our codebase, but would like to know if there are better options.

Reply:
Both are fine